src/OmniSharp.Roslyn.CSharp/Services/Intellisense/CompletionItemExtensions.cs

```diff
@@ -21,9 +21,9 @@ internal static class CompletionItemExtensions
         private const string ParitalMethodCompletionProvider = "Microsoft.CodeAnalysis.CSharp.Completion.Providers.PartialMethodCompletionProvider";
         private const string ProviderName = nameof(ProviderName);
         private const string SymbolCompletionItem = "Microsoft.CodeAnalysis.Completion.Providers.SymbolCompletionItem";
-        private const string SymbolCompletionProvider = "Microsoft.CodeAnalysis.CSharp.Completion.Providers.SymbolCompletionProvider";
         private const string SymbolKind = nameof(SymbolKind);
         private const string SymbolName = nameof(SymbolName);
+        private const string SymbolsKey = nameof(Symbols);
```

Comment:
We already have `Symbols = nameof(Symbols)` below, and it seems that it's not used. Should you use that const instead (or remove it). Both `SymbolsKey` and `Symbols` have the same value.

Reply:
good point - i consolidated them